tests/Type/Symfony/ExtensionTestWithoutContainer.php

```diff
@@ -10,7 +10,15 @@ class ExtensionTestWithoutContainer extends TypeInferenceTestCase
 	/** @return mixed[] */
 	public function dataFileAsserts(): iterable
 	{
-		yield from $this->gatherAssertTypes(__DIR__ . '/data/ExampleControllerWithoutContainer.php');
+		if (class_exists('Symfony\Bundle\FrameworkBundle\Controller\Controller')) {
+			yield from $this->gatherAssertTypes(__DIR__ . '/data/ExampleController.php');
+		}
+
+		if (!class_exists('Symfony\Bundle\FrameworkBundle\Controller\AbstractController')) {
+			return;
+		}
+
+		yield from $this->gatherAssertTypes(__DIR__ . '/data/ExampleAbstractController.php');
```

Comment:
I know that PHPCS forced you to do this but I'd prefer structure that allowed adding new files easily. So I'd suggest adding another dataProvider with a single `class_exists` condition and `yield from` with just a single file. And adding that provider to `testFileAsserts`.

Reply:
Actually I might resolve this myself because I really want to get this released 👍